Scripts/build-ios-framework.sh

```diff
@@ -2,97 +2,89 @@
 
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+set -e
 
 # Load custom build config.
 if [ -r "${SRCROOT}/../.build_config" ]; then
-source "${SRCROOT}/../.build_config"
-echo "MS_ARM64E_XCODE_PATH: " $MS_ARM64E_XCODE_PATH
+  source "${SRCROOT}/../.build_config"
+  echo "MS_ARM64E_XCODE_PATH: " $MS_ARM64E_XCODE_PATH
 else
-echo "Couldn't find custom build config"
+  echo "Couldn't find custom build config"
 fi
 
 # Sets the target folders and the final framework product.
-FMK_NAME=$1
-TGT_NAME=${FMK_NAME}IOS
-FMK_RESOURCE_BUNDLE=${FMK_NAME}Resources
+TARGET_NAME="${PROJECT_NAME} iOS Framework"
+RESOURCE_BUNDLE="${PROJECT_NAME}Resources"
 
-echo "Building ${FMK_NAME} iOS framework."
+echo "Building ${TARGET_NAME}."
 
 # Install dir will be the final output to the framework.
 # The following line create it in the root folder of the current project.
-PRODUCTS_DIR=${SRCROOT}/../AppCenter-SDK-Apple/iOS
-INSTALL_DIR=${PRODUCTS_DIR}/${FMK_NAME}.framework
+PRODUCTS_DIR="${SRCROOT}/../AppCenter-SDK-Apple/iOS"
 
 # Working dir will be deleted after the framework creation.
-WRK_DIR=build
-DEVICE_DIR=${WRK_DIR}/Release-iphoneos
-SIMULATOR_DIR=${WRK_DIR}/Release-iphonesimulator
+WORK_DIR=build
+DEVICE_DIR="${WORK_DIR}/Release-iphoneos/"
+SIMULATOR_DIR="${WORK_DIR}/Release-iphonesimulator/"
 
 # Make sure we're inside $SRCROOT.
 cd "${SRCROOT}"
 
 # Cleaning previous build.
-xcodebuild -project "${FMK_NAME}.xcodeproj" -configuration "Release" -target "${TGT_NAME}" clean
+xcodebuild -project "${PROJECT_NAME}.xcodeproj" -configuration "Release" -target "${TARGET_NAME}" CLANG_ENABLE_MODULE_DEBUGGING=NO  clean 
```

Comment:
Why do we need `CLANG_ENABLE_MODULE_DEBUGGING`?

Reply:
Was part of some fix. Will verify if removing it breaks the CI.